tools/codegen/pkg/interfacegen/testdata/BUILD

```diff
@@ -0,0 +1,84 @@
+package(default_visibility = ["//tools/codegen/pkg/interfacegen:__pkg__"])
+
+load("@org_pubref_rules_protobuf//protobuf:rules.bzl", "proto_compile")
+load("//tools/codegen:generate.bzl", "mixer_proto_library")
+
+mixer_proto_library(
+    name = "list_template_library",
+    testonly = True,
+    protos = ["ListTemplate.proto"],
+)
+
+mixer_proto_library(
+    name = "log_template_library",
+    testonly = True,
+    protos = ["LogTemplate.proto"],
+)
+
+mixer_proto_library(
+    name = "metric_template_library",
+    testonly = True,
+    protos = ["MetricTemplate.proto"],
+)
+
+mixer_proto_library(
+    name = "nested_message_library",
+    testonly = True,
+    protos = ["NestedMessage.proto"],
+)
+
+mixer_proto_library(
+    name = "quota_template_library",
+    testonly = True,
+    protos = ["QuotaTemplate.proto"],
+)
+
+filegroup(
+    name = "generated_files",
+    testonly = True,
+    srcs = [
+        "list_template_library_processor.gen.go",
+        "log_template_library_processor.gen.go",
+        "metric_template_library_processor.gen.go",
+        "nested_message_library_processor.gen.go",
+        "quota_template_library_processor.gen.go",
+    ],
+)
+
+filegroup(
+    name = "golden_files",
+    testonly = True,
+    srcs = [
+        "ListTemplateProcessorInterface.golden.go",
+        "LogTemplateProcessorInterface.golden.go",
+        "MetricTemplateProcessorInterface.golden.go",
+        "NestedMessageProcessorInterface.golden.go",
+        "QuotaTemplateProcessorInterface.golden.go",
+    ],
+)
+
+proto_compile(
```

Comment:
no, we cannot. mixer_proto_library will fail because it tries to build a library. there is no mixer_proto_compile step at the moment. I'm not quite sure what the expected output of mixer_proto_compile should be.  I'd prefer to punt on adding that functionality for now.